easybuild/easyconfigs/b/berkeley_upc/berkeley_upc-2.20.2-GCC-4.9.2.eb

```diff
@@ -0,0 +1,34 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+easyblock = 'ConfigureMake'
+
+name = 'berkeley_upc'
```

Comment:
Seems there are many UPC implementations:
https://www.google.com/search?q=%22other+upc+implementations%22

Do you prefer berkeley_UPC, Berkeley_UPC, berkeley_upc?